qiskit/circuit/library/blueprintcircuit.py

```diff
@@ -32,15 +32,15 @@ class BlueprintCircuit(QuantumCircuit, ABC):
     def __init__(self, *regs, name: Optional[str] = None) -> None:
         """Create a new blueprint circuit.
 
-        The ``_data`` argument storing the internal circuit data is set to ``None`` to indicate
-        that the circuit has not been built yet.
+        The ``_is_built`` argument is set to ``False`` to indicate that the circuit
+        has not been built yet.
```

Comment:
Could we move this to an inline comment, i.e. with `#` instead of in the docstring? I feel like this is not an information we need/want to expose to end level users but rather for developers changing the code.

Reply:
Given the doc string above at the class level and the fact that `self._is_built = False` seems self-explanatory, if it's ok, I just removed this comment in https://github.com/Qiskit/qiskit-terra/pull/7216/commits/d3b12ac0f568ae4a05790d5c6c081995dbe0e4d7.